3rdparty/BUILD

```diff
@@ -124,28 +118,14 @@ jar_library(name='slf4j-api',
               jar(org='org.slf4j', name='slf4j-api', rev='1.6.1'),
             ])
 
-# NB: we have two versions of thrift here due to scrooge requiring one version while apache
-# thrift requires another. This is not usually recommended.
-jar_library(name='libthrift-0.9.2',
+jar_library(name='libthrift',
             jars = [
-              jar(org='org.apache.thrift', name='libthrift', rev='0.9.2')
+              jar(org='org.apache.thrift', name='libthrift', rev='0.10.0')
             ],
             # javac uses undeclared types to build the MessageBuilder API.
             scope='forced',)
 
-target(name='thrift-0.9.2', dependencies = [':libthrift-0.9.2'])
-
-jar_library(name='libthrift-0.6.1',
-            jars = [
-              jar(org='org.apache.thrift', name='libthrift', rev='0.6.1')
-            ])
-
-target(name='thrift-0.6.1',
-       dependencies = [
-         ':commons-lang',
-         ':libthrift-0.6.1',
-         ':slf4j-api',
-       ])
+target(name='thrift', dependencies = [':libthrift'])
```

Comment:
I don't think we need this aliasing any more, it appears to only have existed for 0.6.1 to bring in some extra deps. and for 0.9.2 I guess for symmetry... I'm pretty sure you can just change the name of the jar_library target to `thrift` and be done with it?